aikau/src/main/resources/alfresco/forms/controls/css/TextBox.css

```diff
@@ -18,4 +18,14 @@
 
 .alfresco-forms-controls-TextBox div.control div.dijitValidationTextBox.dijitTextBoxFocused {
    border-color: @focused-border-color;
+}
+
+.alfresco-forms-controls-TextBox.biggerBolder div.control div.dijitValidationTextBox {
+   font-family: @bold-font;
+   font-weight: normal;
+   font-size: @large-font-size;
+}
+
+.alfresco-forms-controls-TextBox.radius div.control div.dijitValidationTextBox {
+   border-radius: @standard-button-border-radius;
```

Comment:
This isn't a button. We probably want an additional LESS variable for form control radius that (along with standard-button-border-radius) should be taken from a root seed variable. It should be possible for example to change a variable to remove the radius from a text box without also removing the radius from a button.